util/ribbon_test.cc

```diff
@@ -411,473 +411,482 @@ TYPED_TEST(RibbonTypeParamTest, CompactnessAndBacktrackAndFpRate) {
   using ConfigHelper =
       ROCKSDB_NAMESPACE::ribbon::BandingConfigHelper<TypeParam>;
 
-  if (sizeof(CoeffRow) < 8) {
+  if constexpr (sizeof(CoeffRow) < 8) {
     ROCKSDB_GTEST_BYPASS("Not fully supported");
     return;
-  }
```

Comment:
To me this is a clear case of early return improving readability by reducing indent depth and apparent complexity. (You want to add indent to 400+ lines of code for this?)

Or does the 4127 warning preclude unreachable code after a constexpr early return? It sounds like a very bad warning. What's wrong with having it disabled globally? If you don't want it disabled in the build why not disable semi-globally in port_win.h or lang.h?